File: app/src/main/java/workshop/akbolatss/tagsnews/model/FeedlyResponse.java
package workshop.akbolatss.tagsnews.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

import workshop.akbolatss.tagsnews.model.dao.RssSource;

public class FeedlyResponse extends BaseResponse {

    @SerializedName("results")
    private List<RssSource> rssSourceList;

    @SerializedName("related")
    private List<String> related;

    public List<RssSource> getRssSourceList() {
        return rssSourceList;
    }

    public List<String> getRelated() {
        return related;
    }
}

File: app/src/main/java/workshop/akbolatss/tagsnews/model/FeedlyResponse.kt
package workshop.akbolatss.tagsnews.model

import com.google.gson.annotations.SerializedName

import workshop.akbolatss.tagsnews.model.dao.RssSource

/**
 * Entity for API response
 */
class FeedlyResponse {

    /**
     * List of Rss sources
     */
    @SerializedName("results")
    val rssSourceList: List<RssSource>? = null

    @SerializedName("related")
    val related: List<String>? = null
}
